vulkaninfo/vulkaninfo.cpp

```diff
@@ -349,7 +349,7 @@ void GpuDumpProps(Printer &p, AppGpu &gpu) {
     if (p.Type() != OutputType::json) {
         if (gpu.inst.CheckExtensionEnabled(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME)) {
             void *place = gpu.props2.pNext;
-            chain_iterator_phys_device_props2(p, gpu.inst, gpu, place, gpu.inst.vk_version);
+            chain_iterator_phys_device_props2(p, gpu.inst, gpu, place, gpu.api_version);
```

Comment:
Shouldn't this be the smaller of the loader/API version and the GPU/driver version?

Using just the API version will do bad things if you have an early loader and a more modern driver.  (Maybe this isn't an expected use case?)